### DISCUSSION TITLE
Midnight Commander over SSH issue

### DISCUSSION THREAD

**Discussion by SolderingArmor** [kovidgoyal/kitty#7862] [Q&A]
Hi! I decided to try out kitty terminal, am I the only one who has trouble with mc over ssh?
Local mc works just fine (with default blue background etc), but when I ssh to my Ubuntu 22.04 server, mc window has black background and is unresponsive (pressing arrow keys does nothing). I've been using terminator and Gnome terminal for years and never had this issue.
I have a default kitty 0.36.1 installation (on Manjaro) with only font changed.

**Comment by kovidgoyal** [comment]
Search the kitty FAQ for ssh.

On Mon, Sep 16, 2024 at 11:18:12AM -0700, Anton Platonov wrote:
> Hi! I decided to try out kitty terminal, am I the only one who has trouble with mc over ssh?
> Local mc works just fine (with default blue background etc), but when I ssh to my Ubuntu 22.04 server, mc window has black background and is unresponsive (pressing arrow keys does nothing). I've been using terminator and Gnome terminal for years and never had this issue.
> I have a default kitty 0.36.1 installation (on Manjaro) with only font changed.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7862
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
